docs/configuration/configuration.md

```diff
@@ -128,7 +128,7 @@ Env file `electron-builder.env` in the current dir ([example](https://github.com
   
   If `react-scripts` in the app dev dependencies, `react-cra` will be set automatically. Set to `null` to disable automatic detection.
 * <code id="Configuration-extraMetadata">extraMetadata</code> any - Inject properties to `package.json`.
-* <code id="Configuration-forceCodeSigning">forceCodeSigning</code> = `false` Boolean - Whether to fail if application will be not signed (to prevent unsigned app if code signing configuration is not correct).
```

Comment:
What's wrong with `will be`?

Reply:
actually, the statement mentions about an action that is `to fail`.

Signing should happen before the process which is going to fail happens

Grammatically, this will be more correct.